.circleci/config.yml

```diff
@@ -135,23 +140,21 @@ workflows:
               only:
                 - "master"
     jobs:
-      - aws_destroy_by_date
+      - aws-destroy-by-date
 
   nightly:
-    triggers:
-      - schedule:
-          cron: "0 5 * * *"
-          filters:
-            branches:
-              only:
-                - master
+#    triggers:
```

Comment:
LGTM once this comment is addressed

Reply:
Though would still be nice to address the comments on the new scripts (copy-paste friendliness of build commands especially)